Lib/unittest/mock.py

```diff
@@ -1583,8 +1565,7 @@ def _get_target(target):
     except (TypeError, ValueError):
         raise TypeError("Need a valid target to patch. You supplied: %r" %
                         (target,))
-    getter = lambda: _importer(target)
-    return getter, attribute
+    return partial(pkgutil.resolve_name, target), attribute
```

Comment:
Is there any reason here to be using `functools.partial` over `lambda`?

Reply:
for consistency. `lambda:` is used twice in this file whereas `partial(` is used six times